php/libraries/UserPermissions.class.inc

```diff
@@ -51,7 +51,7 @@ class UserPermissions
     function select($username)
     {
         // create DB object
-        $DB =& Database::singleton();
+        $DB = \Database::singleton();
```

Comment:
Not relevant to this PR but also.... this is nice.

Reply:
Yeah, it was throwing a database exception on that line and I thought that might be the cause..